searchcore/src/vespa/searchcore/proton/docsummary/isummarymanager.h

```diff
@@ -20,6 +18,8 @@ namespace proton {
 class ISummaryManager
 {
 public:
+    using SummaryConfig = vespa::config::search::internal::InternalSummaryType;
```

Comment:
Consider adding const, to differentiate between config builder (writeable) and config (read only).

Reply:
Fixed